tools/pylint-extensions/azure-pylint-guidelines-checker/README.md

```diff
@@ -53,18 +53,19 @@ You can identify a custom checker by the link to the guidelines included in its
 | C4765 | invalid-use-of-overload | Functions and their overloads must be either all async or all synchronous. | # pylint:disable=invalid-use-of-overload | No Link. | [Example](code_examples.md#invalid-use-of-overload) |
-| C4766 | do-not-log-exceptions | Do not log exceptions in levels other than debug, it can otherwise reveal sensitive information | # pylint:disable=do-not-log-exceptions | [Logging sensitive info](https://azure.github.io/azure-sdk/python_implementation.html#python-logging-sensitive-info) | [Example](code_examples.md#do-not-log-exceptions) |
+| C4766 | do-not-log-exceptions-if-not-debug | Do not log exceptions in levels other than debug, it can otherwise reveal sensitive information | # pylint:disable=do-not-log-exceptions-if-not-debug | [Logging sensitive info](https://azure.github.io/azure-sdk/python_implementation.html#python-logging-sensitive-info) | [Example](code_examples.md#do-not-log-exceptions-if-not-debug) |
 | C4767 | do-not-hardcode-connection-verify | Do not hardcode a boolean value to connection_verify. It's up to customers who use the code to be able to set it | # pylint:disable=do-not-hardcode-connection-verify | No Link. | [Example](code_examples.md#do-not-hardcode-connection-verify) |
 | C4768 | do-not-hardcode-dedent | Do not hardcode dedent value in docstring. Leave the dedent entry in your docstring empty, do not assign a numerical value to it. | # pylint:disable=do-not-hardcode-dedent | No Link. | [Example](code_examples.md#do-not-hardcode-dedent) |
 | C4769 | missing-user-agent-policy | You should include a UserAgentPolicy in your HTTP pipeline. | # pylint:disable=missing-user-agent-policy | [Network operations](https://azure.github.io/azure-sdk/python_implementation.html#network-operations) | [Example](code_examples.md#missing-user-agent-policy) |
 | C4770 | missing-logging-policy | You should include a LoggingPolicy in your HTTP pipeline. | # pylint:disable=missing-logging-policy | [Network operations](https://azure.github.io/azure-sdk/python_implementation.html#network-operations) | [Example](code_examples.md#missing-logging-policy) |
 | C4771 | missing-retry-policy | You should include a RetryPolicy in your HTTP pipeline. | # pylint:disable=missing-retry-policy | [Network operations](https://azure.github.io/azure-sdk/python_implementation.html#network-operations) | [Example](code_examples.md#missing-retry-policy) |
 | C4772 | missing-distributed-tracing-policy | You should include a DistributedTracingPolicy in your HTTP pipeline. | # pylint:disable=missing-distributed-tracing-policy | [Network operations](https://azure.github.io/azure-sdk/python_implementation.html#network-operations) | [Example](code_examples.md#missing-distributed-tracing-policy) |
+| C4773 | do-not-use-logging-exception | Do not use Exception level logging. Use another logging level instead. | # pylint:disable=do-not-use-logging-exception | No Link. | [Example](code_examples.md#do-not-use-logging-exception) |
```

Comment:
Should we add something about this revealing sensitive information like we do with the other rule so people don't just disable